packages/map/lib/src/leaflet/classes/leaflet-map.ts

```diff
@@ -119,13 +119,13 @@ export class LeafletMap {
 	}
 
 	// CENTERING
-	locate() {
+	locate(zoomlevel: number = 19) {
```

Comment:
Don't forget to do a check if the given zoom level is a valid value.
And if you're going to set the default value here as well, it's better to set it in a shared config file.